src/blog/core2.clj

```diff
@@ -0,0 +1,63 @@
+(ns blog.core
+  (:require [clojure.java.io :as io]
+            [markdown.core :as md]
+            [clojure.string :as str]
+            [clojure.instant :as inst]))
+
+;; Set the directory for the blog post markdown files
+(def blog-dir "./blog/")
+
+;; Let's define the HTML that goes above the blog posts html
+(def html-header
+  (slurp (str blog-dir "static/header.html")))
+
+;; Let's define the HTML that goes below the blog posts html
+(def html-footer
+  (slurp (str blog-dir "static/footer.html")))
```

Comment:
As an extension, I suggest using [hiccup](https://github.com/weavejester/hiccup) instead of raw html.

And for CSS, you could try [garden](https://github.com/noprompt/garden) or [girouette](https://github.com/green-coder/girouette) w/ [tailwind styles](https://tailwindcss.com/)